documentation/credscan-process.md

```diff
@@ -0,0 +1,67 @@
+# Guide for monitoring CredScan checks
+
+This guide describes how package owners can monitor their package's Credential Scanner (CredScan) status and correct
+any warnings.
+
+General information about CredScan can be found in the overview documentation at [aka.ms/credscan][credscan_doc]. The
+Azure SDK's motivation and methodology for running CredScan is documented [here][devops_doc].
+
+## Table of Contents
+- [Check CredScan status](#check-credscan-status)
+- [Correct active warnings](#correct-active-warnings)
+  - [True positives](#true-positives)
+  - [False positives](#false-positives)
+
+## Check CredScan status
+
+CredScan is run each night over the entire `azure-sdk-for-js` repository as part of the
+[js - aggregate-reports][aggregate_reports] pipeline. The scan produces a list of active warnings in the "Post
+Analysis" task of the "ComplianceTools" job ([example output][credscan_output]).
+
+Each warning will begin with the path to the file containing a potential credential, as well as the row and column where
+the credential string begins. For example, for a potential credential that starts in row 29 and column 13 of a
+particular file:
+```
+##[error]sdk/{service}/{package}/{path-to-file}.ts:sdk/{service}/{package}/{path-to-file}.ts(29,13):
+```
+
+The warning will then list an error code and description of why the potential credential was flagged.
+
+## Correct active warnings
+
+If you find any warnings listed for a package you own, the next step is to determine if the potential credential found
+by CredScan is an actual credential (a true positive), or a fake credential/false flag (a false positive).
+
+### True positives
+
+If CredScan discovers an actual credential, please contact the EngSys team at azuresdkengsysteam@microsoft.com so any
+remediation can be done.
```

Comment:
Also, add regarding re-recording if the true positive is from a test recording if it applies?

Reply:
We will enable this in PR validation soon. The recordings will be blocked too so I don't want to special case it in this document.